docs/install.sh

```diff
@@ -1,15 +1,14 @@
 #!/bin/bash
-mkdir -p /imaging/{bin,conf/rdmp,data} /imaging/db/{mongo,sql} 
-curl -sL https://github.com/HicServices/RDMP/releases/download/v8.0.7/rdmp-8.0.7-cli-linux-x64.tar.xz | tar x --strip-components=1 -JC /imaging/bin/rdmp -f - 
-curl -sL https://github.com/SMI/SmiServices/releases/download/v5.3.0/smi-services-v5.3.0-linux-x64.tgz | tar x --strip-components=1 -zC /imaging/bin/smi -f - 
-mysql --uroot --password=mysqlrootpassword < schema.sql
+mkdir -p /imaging/{bin/rdmp,bin/smi,conf/rdmp,data} /imaging/db/{mongo,sql} 
+curl -sL https://github.com/HicServices/RDMP/releases/download/v8.1.1/rdmp-8.1.1-cli-linux-x64.tar.xz | tar x --strip-components=1 -JC /imaging/bin/rdmp -f - 
+curl -sL https://github.com/SMI/SmiServices/releases/download/v5.4.0/smi-services-v5.4.0-linux-x64.tgz | tar x --strip-components=1 -zC /imaging/bin/smi -f - 
+mysql --user=root --password=mysqlrootpassword < schema.sql
```

Comment:
The schema makes some assumptions about something called an "SmiSqlPassword" used in an IDENTIFY BY line. 
It isn't explained what this is, or how to set it in the instructions

Reply:
Documented now - will try to automate at some point later. (Target use is demo and/or isolated environment, so not normally an issue.)

Reply:
Thanks, this clears it up. I'm still having issues with the grants in the schema causing it to fail, but that might just be because I've already granted the smi mysql user admin rights

Reply:
Yes, the script assumes it has a fresh VM to install to and sets up the new user accordingly, having an existing one will probably be an issue.